Removed restrictions and fixed other minor issues.

kreinold opened the pull request:
[DM-505](https://broadworkbench.atlassian.net/browse/DM-505)

rsc3 commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -1694,11 +1644,11 @@ TerraCore:hasAnatomicalRegion
-    );
-  ];
+      owl:unionOf (
+          xsd:anyURI
```
Repeated several times below

kreinold replied:
The reason for [anyURI ](http://books.xmlschemata.org/relaxng/ch19-77009.html) is that this data type imposes restrictions on xsd:string which allow some validation to be performed if we know we have a URI.  See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

rsc3 commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -2296,11 +2242,11 @@ TerraCore:hasHostOrganism
-    );
-  ];
+      owl:unionOf (
+          obo:OBI_0100026
```
Can't unionOf object and datatype

kreinold replied:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

rsc3 commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -2791,11 +2737,11 @@ TerraCore:hasSequenceLocation
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
Can't unionOf object with datatype

kreinold replied:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

rsc3 commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -3112,11 +3058,11 @@ TerraCore:usesAntibody
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
can't unionOf object with datatype

kreinold replied:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

rsc3 commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -3142,13 +3088,13 @@ TerraCore:usesReferenceAssembly
-  ];
+  rdfs:range [
+      owl:unionOf (
+          xsd:anyURI
```
can't unionOf datatypes and object

rsc3 commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -3260,11 +3181,11 @@ TerraDCAT_ap:hasOwner
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
can't unionOf datatype and object

kreinold replied:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

kreinold commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -1559,11 +1509,11 @@ TerraCore:confirmsDisease
-      xsd:anyURI
-    ) ;
-  ];
+      owl:unionOf (
```
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

kreinold commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -2815,11 +2761,11 @@ TerraCore:hasServiceProvider
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

kreinold commented on src/terra-core/TerraCoreDataModel.ttl:
```diff
@@ -1830,13 +1780,13 @@ TerraCore:hasChild
+      owl:unionOf (
+          TerraCore:Donor
+          TerraCore:FamilyMember
+          TerraCore:HumanDonor
```
Fix in additional checkin.

kreinold commented:
PR has been updated with the following:
•	Removed foaf equivalent classes; no longer used. 
•	Fixed typo in definition
•	hasAssayType should be an Object Property
•	Removed HumanDonor from range where Donor was already specified. (per comment)
